a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -1125,30 +1120,31 @@ <h4>Examples</h4>
 					<ul>
 						<li data-localization-id="pre-recorded-audio-only"
 							data-localization-mode="descriptive">
-							Audiobook with no text alternative</li>
-						<li data-localization-id="pre-recorded-audio-complementary"
+							Prerecorded audiobook with no text alternative</li>
+						<!-- <li data-localization-id="pre-recorded-audio-complementary"
 							data-localization-mode="descriptive">
 							Contents available as complementary audio
 							and
-							text</li>
+							text</li> -->
 						<li data-localization-id="pre-recorded-audio-synchronized"
 							data-localization-mode="descriptive">All the
 							content is available as prerecorded audio
 							synchronized with text</li>
+<li data-localization-id="pre-recorded-audio-no-metadata"
+							data-localization-mode="descriptive">
+							Prerecorded audio not provided</li>
 					</ul>
 				</aside>
 
 				<aside class="example" title="Compact explanations">
 					<ul>
 						<li data-localization-id="pre-recorded-audio-only"
-							data-localization-mode="compact">Audio
-							only</li>
-						<li data-localization-id="pre-recorded-audio-complementary"
-							data-localization-mode="compact">
-							Complementary audio and text</li>
+							data-localization-mode="compact">Prerecorded audiobook only</li>
 						<li data-localization-id="pre-recorded-audio-synchronized"
+							data-localization-mode="compact">Prerecorded synchronized audio and text</li>
```

Comment:
This may be intentional to call out "prerecorded synthetic" as opposed to "prerecorded human voice" but I'm not sure. @GeorgeKerscher which did you intend to say?